README.md

```diff
@@ -0,0 +1,26 @@
+# Field Formatter
+
+[Field Formatter](https://www.drupal.org/project/field_formatter) module adds formatters for entity reference fields to output only a specific field or field value using a specific formatter w/o needing to configure media view mode and configuring it to display properly. This module provides two formatters: *Field formatter with inline settings* and *Field formatter from view display*.
```

Comment:
This reads a bit strange. I'd made first sentence shorter. What about skipping everything after "... output only a specific field"